src/modules/decentralized-web-node/entities/message.entity.ts

```diff
@@ -0,0 +1,52 @@
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+import { CollectionsQueryDescriptorDto } from '../dtos/collections-query-descriptor.dto';
+import { CollectionsWriteDescriptorDto } from '../dtos/collections-write-descriptor.dto';
+import { MessageDto } from '../dtos/message.dto';
+
+@Entity()
+export class Message {
+  static create(data: Partial<Message>): Message {
+    const entity = new Message();
+    Object.assign(entity, data);
+    return entity;
+  }
+
+  @PrimaryColumn()
+  objectId: string;
+
+  @Column()
+  createdBy: string;
+
+  @Column({ nullable: true })
+  data?: string;
+
+  @Column('jsonb')
+  message: MessageDto;
+
+  @Column('jsonb')
+  descriptor: CollectionsQueryDescriptorDto | CollectionsWriteDescriptorDto;
+
```

Comment:
`descriptor` shouldn't be stored, I made that by mistake. For message I used this interface bc we are saving exactly what comes from user to database.

Reply:
Ah okay, you're thinking that is best to have a kind of "backup" of the message in the `message` property? Or I guess it is handy to have because it is what we return in response to a `Query`?

Reply:
The second option: `it is handy to have because it is what we return in response to a Query`